runtime/binding-asyncapi/src/main/java/io/aklivity/zilla/runtime/binding/asyncapi/internal/config/composite/AsyncapiCompositeGenerator.java

```diff
@@ -495,7 +495,7 @@ protected final void injectPayloadModel(
                                 .name("catalog0")
                                 .schema()
                                     .version("latest")
-                                    .subject(subject)
+                                    .subject(message.payload.name)
```

Comment:
This should be consistent with the others, either `subject` everywhere or `message.payload.name` everywhere.

The value of `subject` is passed as part of the URL to access schema registry.
```
/subjects/{0}/versions/{1}
```
So this should probably be based on the channel address topic with `-value` suffix, rather than relying on any naming convention alignment between message payload name and channel address (kafka topic).

Note: given that channels (kafka topics) can have more than one message type in the general case, seems like we actually want to potentially use topic-record name strategy instead of topic name strategy to differentiate the schemas for different channel (topic) messages.

In that case, it  would seem to make sense to base the record part of the topic-record name strategy on the message name within the channel messages.